rust_snuba/src/processors/mod.rs

```diff
@@ -9,6 +11,7 @@ type ProcessingFunction =
 pub fn get_processing_function(name: &str) -> Option<ProcessingFunction> {
     match name {
         "QuerylogProcessor" => Some(querylog::process_message),
+        "ProfilesProcessor" => Some(profiles::process_message),
```

Comment:
This will probably change in the future but right now this name should match https://github.com/getsentry/snuba/blob/master/snuba/datasets/processors/profiles_processor.py#L15 since it swaps out the Python processor for a Rust one if it finds one matching the class name.

```suggestion
        "ProfilesMessageProcessor" => Some(profiles::process_message),
```